R/addLayer.R

```diff
@@ -26,8 +26,7 @@ addLayer = function(filename, ...) {
   response <- POST(url=layerUrl, body=rawData, 
                    content_type('application/octet-stream'),
                    add_headers('Authorization' = getAuth(), 'Connection'='keep-alive'))
-  
-  if (http_status(response)$category == "Success") {
+  if (http_status(response)$category == "success" | http_status(response)$category == "Success") {
```

Comment:
Zack mentioned this in standup too, I'll do that. thanks. :) Not sure why this came back again, but dang it's annoying...